src/main/java/com/bravo/user/dao/model/User.java

```diff
@@ -3,21 +3,24 @@
 import com.bravo.user.model.dto.UserSaveDto;
 import java.time.LocalDateTime;
 import java.util.UUID;
-import javax.persistence.Column;
-import javax.persistence.Entity;
-import javax.persistence.Id;
-import javax.persistence.Table;
-import lombok.Data;
+import javax.persistence.*;
+
+import lombok.*;
 
 @Entity
-@Data
+@Getter
+@Setter
+@AllArgsConstructor
```

Comment:
With lombok not generating toString, Equals and Hashcode;  what is the desired effect from this change?

Reply:
Generally, I only use the Data annotation in model objects cause it can cause performance and memory issues as seen in the screenshot below.  It's not really noticeable when the entities don't have any join constrains, but as entity relationships are built out, this can cause issues.  

I could have left the Data tag and added the ToString.Exclude tag to the OneToOne relationship I added for user and auth, but I generally use Getter and Setter along with the toString, Equals and Hashcode as needed on entities (It's a preference, but I'm fine with either approach)
<img width="716" alt="data-tag" src="https://user-images.githubusercontent.com/27737590/131399222-8359bc0f-8cbf-48a1-a0be-6904d3b169c8.png">